src/Steps/Release/UpdateTranslations.php

```diff
@@ -221,6 +239,31 @@ protected function pullSource(OutputInterface $output, $modules)
     }
 
     /**
+     * Tidy yaml files using yamlclean ruby gem
+     *
+     * @param OutputInterface $output
+     * @param Module[] $modules List of modules
+     */
+    protected function cleanYaml(OutputInterface $output, $modules)
+    {
+        foreach ($modules as $module) {
+            $name = $module->getName();
+            $this->log(
+                $output,
+                "Cleaning YAML sources for <info>{$name}</info>"
+            );
+
+            foreach (glob($module->getLangDirectory()."/*.yml") as $sourceFile) {
+                $cleanCommand = sprintf(
+                    '(yamlclean %1$s > %1$s.new && mv %1$s.new %1$s)',
+                    $sourceFile
```

Comment:
I’m pretty sure this is something to do with the order in which bash opens/truncates files when redirecting output with `>`.

Another (probably nicer) alternative that works is:
```bash
echo "$(yamlclean %$1s)" > %$1s
```